openeogeotrellis/geopysparkdatacube.py

```diff
@@ -572,9 +572,7 @@ def partitionByKey(spatialkey):
     def chunk_polygon(
         self,
         reducer: Union[ProcessGraphVisitor, Dict],
-        # TODO: it's wrong to use MultiPolygon as a collection of polygons. MultiPolygons should be handled as single, atomic "features"
-        #       also see https://github.com/Open-EO/openeo-python-driver/issues/288
-        chunks: MultiPolygon,
+        chunks: DriverVectorCube,
```

Comment:
This changes an API signature defined in openeo-python-driver. I don't think there is PR for that already

Reply:
There is, you can find it here: https://github.com/Open-EO/openeo-python-driver/pull/291